apps/backend-e2e/eslint.config.mjs

```diff
@@ -0,0 +1,13 @@
+import baseConfig from '../../eslint.config.mjs';
+
+export default [
+  {
+    ignores: ['**/dist'],
+  },
+  ...baseConfig,
+  {
+    files: ['**/*.ts', '**/*.tsx', '**/*.js', '**/*.jsx'],
+    // Override or add rules here
+    rules: {},
+  },
```

Comment:
Can we remove all this boilerplate? It seems to be a placeholder without function.